### PR TITLE
Consistency fixes for RPC descriptions

### PR DESCRIPTION
**Pull request by ch4ot1c**

### CODE REVIEW

File: src/rpc/blockchain.cpp [hunk 1]
```diff
@@ -509,17 +509,17 @@ static UniValue getmempoolancestors(const JSONRPCRequest& request)
 {
     if (request.fHelp || request.params.size() < 1 || request.params.size() > 2) {
         throw std::runtime_error(
-            "getmempoolancestors txid (verbose)\n"
+            "getmempoolancestors txid ( verbose )\n"
```
Comment (practicalswift) [inline src/rpc/blockchain.cpp]: What is the original rationale behind the spacing in `( verbose )`?

Comment (ch4ot1c) [reply src/rpc/blockchain.cpp]: Unsure. Maybe to differentiate it from other normal options. It was also done with `( verbosity )`, so I matched it.

**Review by jimmysong — COMMENTED** [review]
It would be nice to get all the capitalizations consistent as well. Sometimes the description for a hash result will say "The block hash" others will say "the block hash".

**Comment by MarcoFalke** [comment]
The formatting fixes would be more elegantly solved with a helper class that takes care of the formatting: See #14378

**Comment by ch4ot1c** [comment]
Also:
- First letter capitalization consistency
- Final `.` consistency

**Comment by fanquake** [comment]
Please squash your commits, and we can merge this in. Moving forward we're likely to be generating this kind of documentation, so that styling can remain consistent.

**Comment by ch4ot1c** [comment]
Squashed

**Comment by fanquake** [comment]
utACK b8edb98

**Comment by MeshCollider** [comment]
utACK https://github.com/bitcoin/bitcoin/pull/14373/commits/b8edb9810a699015e997e2098dddb2a6cfacbed6

**Comment by promag** [comment]
It would be nice to have a plan for this consistency/cleanup/format etc. There is nothing in place to prevent adding the same inconsistencies. As it is I'm -0.

**Comment by Sjors** [comment]
utACK b8edb98

@promag this seems like a strict improvement even without a plan. However I wouldn't want to review these kinds of pull requests too often, so looking forward to @achow101's work.

**Comment by promag** [comment]
Alrighty then 👍

**Comment by promag** [comment]
Missed `"listtransactions (dummy count skip include_watchonly)\n"`?

**Pull request merged**